tests/unit/config_tests.py

```diff
@@ -20,6 +20,8 @@ class TestConfigParser(unittest.TestCase):
         ('--keep-builddir', 'keep_builddir', True),
         ('--build-versions-repository-url=foo', 'build_versions_repository_url', 'foo'),
         ('--build-version=foo', 'build_version', 'foo'),
+        ('--mock-args=--enable-plugin=tmpfs --plugin-option=tmpfs:keep_mounted=True', 'mock_args',
```

Comment:
@fabianorosas not sure I get what you mean. Are you saying that testing with explicit mock arguments might cause someone to think those arguments will be parse as well?

If so, I agree with you.

For example:

Is `--plugin-option=tmpfs:keep_mounted=True` a parameter given to mock? Or is it a parameter parsed by the host os config parser? It is a bit confusing. Shouldn't the mock-args parameters be surrounded by quotes, since they are separated by spaces?